tools/dgeni/karma.conf.js

```diff
@@ -0,0 +1,31 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+
+module.exports = function (config) {
```

Comment:
I don't think we run tests in the browser, so I don't think this file does anything?

Reply:
you're right. Removed